fast64_internal/sm64/sm64_objects.py

```diff
@@ -2156,6 +2157,10 @@ def draw_export_options(self, layout):
             col.prop(self, "export_bhv")
             self.draw_obj_name(layout)
 
+    @property
+    def actor_names(self) -> list:
+        return list(dict.fromkeys(filter(None, [self.obj_name_col, self.obj_name_gfx])).keys())
```

Comment:
I assume the dict keys business is for de-duplication?
Simpler:
```suggestion
        return list(set(filter(None, [self.obj_name_col, self.obj_name_gfx])))
```

Reply:
This can change order

Reply:
Sadge ok then